widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -223,11 +223,11 @@ internal class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: In
         companyName: String?,
         queueId: String?,
         visitorContextAssetId: String?,
-        useOverlays: Boolean = false,
+        useOverlays: Boolean? = null,
         screenSharingMode: ScreenSharing.Mode? = null,
         chatType: ChatType = ChatType.LIVE_CHAT
     ) {
-        Dependencies.getSdkConfigurationManager().isUseOverlay = useOverlays
+        useOverlays?.also { Dependencies.getSdkConfigurationManager().isUseOverlay = it }
```

Comment:
Hm, so engagement value for `useOverlays` will override the SDK initialisation value. 🤔

Reply:
Yes!

Nothing has changed in this sense.

Reply:
Ah, indeed, you are right.